samples/IdentityServer.ServerSample/Controllers/AccountController.cs

```diff
@@ -46,11 +47,16 @@ public IActionResult ExternalLogin(string provider, string returnUrl)
                 RedirectUri = Url.Action(nameof(ExternalLoginCallback)),
                 Items =
                 {
-                    {"returnUrl", returnUrl},
-                    {"scheme", provider}
+                    { "returnUrl", returnUrl },
+                    { "scheme", provider },
                 }
             };
 
+            if (provider.Equals(BankIdAuthenticationDefaults.SameDeviceAuthenticationScheme))
```

Comment:
I would always set this (`{ "cancelReturnUrl", .... },`), makes it cleaner and you do do the check later on wheter or not to use it.